trainee/djtraining.tex

```diff
@@ -309,8 +309,8 @@ \subsection{Suspensions}
 Suspendable offenses include:
 
 \begin{itemize}
-    \item Improper sign-off/sign-on.  Do not leave the station without
-        automation running.
+    \item Improper sign-off/sign-on.
+    \item Leaving the station without automation running. 
```

Comment:
Technically there's trailing whitespace here, but I'm not going to hold it up over that.